build/azure-pipelines.yml

```diff
@@ -67,10 +67,13 @@ stages:
   - template: ../src/cli/.azuredevops/stage-publish-cli.yml
 
 #-endif
+- stage: Commit_Validation
+  dependsOn: []
+  jobs:
+    - template: stage-commit-validation.yaml
+    
 - stage: Build_Staging
-#-if false
-  dependsOn: [] # This removes the implicit dependency on previous stage and causes this to run in parallel.
-#-endif
+  dependsOn: Commit_Validation
```

Comment:
The `CopyTool_GeneratedApp` stage must also depend on the `Commit_Validation` stage to avoid useless build time.
![image](https://github.com/user-attachments/assets/a8432ead-c843-4501-8225-18814e9ea511)